### DISCUSSION TITLE
Corner use case: Tell Sonarr to delete based on a list?

### DISCUSSION THREAD

**Discussion by douginoz** [meisnate12/Plex-Meta-Manager#191] [Q&A]
I incorrectly used a themoviesdb.org list as a Christmas list but it wasn't.  Now I have about 400 new movies that I don't want because they're not Christmas.  
Is there any way to configure PMM to use that list to delete, like a sync_mode that does a pure delete?

**Comment by meisnate12** [comment]
no PMM doesn't have any ability like that

**Reply by douginoz** [reply]
Probably a good idea not to!
I've cleaned things up an alternate way.  I'll document for others in the future that do this same sort of "oops"/
In Radarr v3, sort movies by date added, the most recent at the top.
Create a custom filter.  In my case, I created a filter that included any movie with the path /blah/blah/Christmas (where all my Christmas movies go).
This should now display everything in that folder, sorted most recent first.
I then manually went through the list in Movie Editor mode, selecting all the ones that had been added by the list that I added into PMM.  These were mostly clumped together so I could select the first in the list then hold shift, and select the last, to select all of them.    There were a few scattered elsewhere that I repeated this selection method for until I had gone through enough of the list to be sure that I'd got them all.  Anything that had been added a week ago, for instance, wouldn't have been because of the mistaken PMM run I did yesterday.
After selecting the movies, I used Delete and told it to delete the movies on disk as well.
Finally, I modified my PMM "Christmas Movies.yml" config file to remove the incorrect list reference.

**Comment by douginoz** [comment]
Sorry, hope I'm not spamming.  I don't need/expect immediate replies so don't feel that I'm pressuring you!

Is the following possible?  It doesn't actually work (yet) as-is but there might be another way to accmplish the goal.
 
```
collections: 
Hallmark Christmas:
    tmdb_list:
      - https://www.themoviedb.org/list/6252
      add_to_arr: false
    imdb_list:
      - https://www.imdb.com/list/ls062518819/
    add_to_arr: true

```
 I want to have a "Hallmark Christmas" collection that gets checked and updated.  The tmdb list here is _all_ Hallmark movies, Christmas included.  I don't want everything in that list, but I still want PMM to check that list and move any new movies into it, in case I happen to add a movie in the future that's in that list.  So I want to "add_to_arr: false".   But I also have the imdb list that _is_ Hallmark movies, and I want those to be included in this collection.

**Reply by meisnate12** [reply]
i would probably just leave the collection `radarr_add: false` and then just add the imdb list as an imported list for radarr to check for movies. 

it would be a lot to move `radarr_add` to the builder level because every builder would need to support it
